tasks/queue-granules/index.js

```diff
@@ -113,13 +113,13 @@ async function queueGranules(event, testMocks = {}) {
             ),
             granuleId: queuedGranule.granuleId,
             status: 'queued',
-            createdAt: queuedGranule.createdAt ? queuedGranule.createdAt : createdAt,
+            createdAt,
```

Comment:
This doesn't work -- we *don't* want the granule to be updated with a *new* createdAt time if it already has one, as the Queued record will wind up with a *later* time stamp than the downstream record. 

I think you mean queuedGranule.createdAt?

Reply:
Interesting because yes.  it passed the tests when I did that.
But yes that would have to be queuedGranule.createdAt because that is what was updated in the updateGBCA function.  Ok, in that case this isn't that much cleaner I suppose